src/gnatss/posfilter/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .posfilter import kalman_filtering, spline_interpolate, rotation
+
+__all__ = ['kalman_filtering', 'spline_interpolate', 'rotation']
```

Comment:
Please modify the imports to utilize this new import change.
e.g. `src/gnatss/posfilter/run.py`
```
from . import kalman_filtering, rotation, spline_interpolate
```

Reply:
Sure I'll create another issue about this.